process-engine/skeleton
# Feature/minstall 3

## What did you change?

I added a `package.json`, so the installation with minstall 3 can be simplified. I also updated some dependencies

## How can others test the changes?

check that the meta-setup still works, even in its simplified form [here](https://github.com/process-engine/process_engine_meta/blob/f3e250d33c477af3bc4cee4fd88907aa19cfe4b4/setup.sh)

## PR-Checklist

Please check the boxes in this list after submitting your PR:
